### PR TITLE
Allow filtering log messages per level (#400)

### PR DESCRIPTION
**Pull request by fabioDMFerreira**
* Add logger;

* Add command flags "show-error" and "show-warnings" to every command;

* Append logger to context;

### CODE REVIEW

**Comment by fabioDMFerreira** [comment]
In this PR, we append a logger utility to the context. This way we can use it in every ops command.

The exceptions are the volumes operations which don't have access to a context (https://github.com/nanovms/ops/issues/763).

With this, we can filter the messages printed in the user console (https://github.com/nanovms/ops/issues/764).

File: cmd/appendGlobalCmdFlagsToConfig.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package cmd
+
+import (
+	"github.com/nanovms/ops/lepton"
+	"github.com/spf13/pflag"
```
Comment (eyberg) [inline cmd/appendGlobalCmdFlagsToConfig.go:5]: is it necessary to have another flag lib? we already have one in addition to the existing stdlib stuff

File: lepton/logger_test.go [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/nanovms/ops/lepton"
+)
+
+const (
+	colorRed    = "\033[31m"
```
Comment (eyberg) [inline lepton/logger_test.go:11]: why are these repeated from lepton/logger?

**Comment by eyberg** [comment]
the only concern here is the use of yet another flag lib - it's not possible to consolidate?

**Review by eyberg — APPROVED** [review]

File: cmd/appendGlobalCmdFlagsToConfig.go (hunk 1 above)
Comment (fabioDMFerreira) [reply cmd/appendGlobalCmdFlagsToConfig.go:5]: This is an internal library used by `spf13/cobra`. We're using to receive `cmd.Flags()` result in a function.

File: lepton/logger_test.go (hunk 2 above)
Comment (fabioDMFerreira) [reply lepton/logger_test.go:11]: didn't make sense to me making these constants values readable from external packages.

**Pull request merged**